thcrap_test/src/win32_utf8.cpp

```diff
@@ -18,69 +18,6 @@ std::filesystem::path get_dll_source_path()
 	return exe_path;
 }
 
-TEST(NewUTF8Test, UTF8_TO_UTF16) {
-
-#define MakeUTFStrings(string) \
-	const char utf8_in[] = u8 ## string;\
-	const char16_t utf16[] = u ## string;\
-	const char32_t utf32[] = U ## string;
-
-	//MakeUTFStrings("Iñtërnâtiônà£ißætiøn☃💩");
-	//MakeUTFStrings("e💩🍙e");
-
-	const char utf8_in[] = u8"e💩🍙e";
-	const char16_t utf16[] = u"e💩🍙e";
-	const char32_t utf32[] = U"e💩🍙e";
-
-	const char16_t* utf16_out_bit_test_reset = utf8_to_utf16_bit_test_reset(utf8_in);
-	const char16_t* utf16_out_masking = utf8_to_utf16_masking(utf8_in);
-	const char16_t* utf16_out_bit_scan = utf8_to_utf16_bit_scan(utf8_in);
-	//const char32_t* utf32_out = utf8_to_utf32(utf8_in);
-
-	printf("\nConverted UTF8 Bytes: (Bit Test And Reset)\n");
-	for (size_t i = 0; i < elementsof(utf16); ++i) {
-		printf("%04hX", (uint32_t)utf16_out_bit_test_reset[i]);
-	}
-	
-	printf("\nConverted UTF8 Bytes: (Masking)\n");
-	for (size_t i = 0; i < elementsof(utf16); ++i) {
-		printf("%04hX", (uint32_t)utf16_out_masking[i]);
-	}
-
-	printf("\nConverted UTF8 Bytes: (Bit Scan)\n");
-	for (size_t i = 0; i < elementsof(utf16); ++i) {
-		printf("%04hX", (uint32_t)utf16_out_bit_scan[i]);
-	}
-	
-	printf("\nCorrect UTF16 Bytes:\n");
-	for (size_t i = 0; i < elementsof(utf16); ++i) {
-		printf("%04hX", (uint32_t)utf16[i]);
-	}
-
-	/*printf("\nConverted UTF8 Bytes:\n");
-	for (size_t i = 0; i < elementsof(utf32); ++i) {
-		printf("%08X", utf32_out[i]);
-	}
-
-	printf("\nCorrect UTF32 Bytes:\n");
-	for (size_t i = 0; i < elementsof(utf32); ++i) {
-		printf("%08X", utf32[i]);
-	}*/
-
-
-	//WCHAR_T_DEC(utf8_in);
-	//WCHAR_T_CONV(utf8_in);
-	/*printf("WChar Output Bytes:\n");
-	for (size_t i = 0; i < elementsof(utf16); ++i) {
-		printf("%04hX", utf8_in_w[i]);
-	}*/
-	//WCHAR_T_FREE(utf8_in);
-	EXPECT_TRUE(
-		!wcscmp((wchar_t*)utf16, (wchar_t*)utf16_out_masking) &&
-		!wcscmp((wchar_t*)utf16, (wchar_t*)utf16_out_bit_scan)
-	);
-}
-
```

Comment:
Why don't we keep this test for utf8_to_utf16?

Reply:
Because it was made to check all 3 different functions are correct. I re-introduce the old utf8_to_utf16 with the mbrtoc16 call in just the test or use string literals to still have tests for utf8_to_utf16 and utf8_to_utf32